repository-data/webfiles/src/main/resources/site/freemarker/common/white-space.ftl

```diff
@@ -0,0 +1,16 @@
+<#ftl output_format="HTML">
+
+<#if size??>
+    <#if size == "Small">
+        <div class="nhsd-!t-padding-bottom-3"></div>
+    </#if>
+    <#if size == "Medium">
+        <div class="nhsd-!t-padding-bottom-6"></div>
+    </#if>
+    <#if size == "Large">
+        <div class="nhsd-!t-padding-bottom-9"></div>
+    </#if>
+    <#if size == "Extra Large">
+        <div class="nhsd-!t-padding-bottom-10"></div>
```

Comment:
I am not sure that CSS class is legal https://www.w3.org/TR/CSS2/syndata.html#:~:text=In%20CSS%2C%20identifiers%20(including%20element,hyphen%20followed%20by%20a%20digit.

Reply:
The '!' is to indicate an override token (which you can see examples of, in the gov UK design system https://design-system.service.gov.uk/styles/spacing/)

Reply:
Nice okay